src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -454,8 +454,14 @@ export class OperationDetails {
     }
 
     private scrollToOperation() {
```

Comment:
I think we should rename this method, since it's not used only for scrolling to the operation, but rather any scrolling inside the operation details. What do you think?